### ISSUE TITLE
Combine transition glasses and sunglasses

### ISSUE THREAD

**Issue by Agnistel** [CleverRaven/Cataclysm-DDA#76790]
### Is your feature request related to a problem? Please describe.

Except for the slightly different exchange price, transition glasses and sunglasses are indistinguishable from each other, but they create confusion and visual noise due to the clutter of identical items. Especially considering the separate name for stylish items.

![image](https://github.com/user-attachments/assets/e9d844b4-c56b-40cc-ac9f-cb533094adf0)
![image](https://github.com/user-attachments/assets/84b17bea-dd72-441e-9752-dbf8d954152b)

### Solution you would like.

Remove transition glasses or make them stylish but remove the stylish sunglasses

**Comment by Psithief** [comment]
I would want sunglasses to be in some way inferior to transition glasses, but constantly needing to take them off and put them on to avoid vision penalties outside full daylight is too much realism.

**Comment by fairyarmadillo** [comment]
Transition lenses don't instantly transition irl. They take about 30 seconds to darken when exposed to bright sunlight (is it slower when it's less bright?) but take several minutes to lose their tint in the dark. This should definitely affect gameplay. I think the effect would need to be hardcoded, unless I'm underestimating EOCs here.

**Comment by GuardianDll** [comment]
i don't think it is worth emulating it imo, it doesn't matter much if player gonna get glare immune 30 seconds after leaving the building (since glare effect itself barely affect anything), and amount of tracking it require is enormous
